src/test/java/ch/liquidmind/inflection/compiler/InflectionCompilerTest.java

```diff
@@ -0,0 +1,55 @@
+package ch.liquidmind.inflection.compiler;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import ch.liquidmind.inflection.compiler.CompilationJob.CompilationMode;
+import ch.liquidmind.inflection.compiler.util.CompilerTestUtility;
+
+public class InflectionCompilerTest {
```

Comment:
Need to format the code using "standard" ZHAW IWI beautifier; I'm sending you the eclipse files now. Also: could you see about integrating this with gradle? I know that there is a plugin that reads eclipse formatter files---Alpnet used it in ExpAct, but it wasn't working, so I took it out. I assume it does, however, work if configured correctly.